docs/migration-guides/0.8-0.9.md

```diff
@@ -0,0 +1,37 @@
+## Migrate from v0.8 to v0.9
+
+Kyma 0.9.0 release ships completely new version of the solution for handling content inside Kyma. The feature is called Headless CMS and it introduces content management through custom resources. 
+
+> TIP:For more details read release details in [GitHub](https://github.com/kyma-project/kyma/releases/tag/0.9.0) and [Kyma Blog](https://kyma-project.io/blog/).
+
+This change influences the Application Connector component, services specifications persistence part. This change has impact on Kyma users that upgrade from 0.8.0 to 0.9.0 because the old location of the specifications is not supported by Application Connector.
+To make migration as seamless as possible we introduce an automated migration script that does two things:
+- removes old and registers new custom resources for Headless CMS and Asset Store components
+- migrates specifications registered with Application Connector prior 0.9.0 into Headless CMS
+> TIP:For more details read migration [script](https://github.com/kyma-project/kyma/blob/master/resources/cms/templates/_helper_migrate_to_cms.txt).
+
+### Migration script limitations
+
+Migration script does not address all the possible use cases. Its scope is focused only on the use cases that we know are used by Kyma users.
+This means the script assumption is:
+- One service has only one specification. There are no services with two different specs, like for example OpenAPI and AsyncAPI
+- Documentation registration exposed through Metadata service is not used in current integrations.
+
+The script basically takes specs from registered applications from location like `https://minio.{$DOMAIN}/content/service-class/{$SERVICE_CLASS_ID}/{$SPEC_NAME}.json` (where spec name is `asyncApiSpec` or `apiSpec`)
+and saves them as `ClusterDocsTopic` custom resource.
+
+### Manual migration
+
+Uncovered use cases require manual migration after a successful upgrade. Please familiarize with documentation for Headless CMS component to understand how to use it.
+Keep in mind that prior 0.9.0 documentation was persisted in a single `content.json` file. Now, one of the options is to deliver content from `content.json` as separate files in a package.
+
+In case of any issues with the migration please contact us on [Slack](http://slack.kyma-project.io) on `core-and-supporting` channel.
```

Comment:
```suggestion
In case of any issues with the migration, contact us on [Slack](http://slack.kyma-project.io) on the `core-and-supporting` channel.
```